### DISCUSSION TITLE
Diseño App

### DISCUSSION THREAD

**Discussion by dani97munoz** [cometa-dani/cometa-mobile-app-ui#4] [General]
Aquí adjunto uno de los diseños que había hecho para la app. Puedes tomarlos en en cuenta a la hora de diseñar.

Gracias!
[Uploading protoio_Cometa_Social_App_1.pdf…]()

**Comment by dani97munoz** [comment]
Aquí también tienes la paleta de colores y el logo. 

![1 copy-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/1f4fdad7-203d-46f0-8671-6799f0680b30)
![1-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/2ced6585-ae57-4880-a11f-358808335cde)
![2-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/d7870acf-6773-4fde-ab0a-72c6ff64ffac)
![22-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/bc2efb9f-6217-4a60-b849-d953ebc91359)
![Artboard 1 copy 6-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/ab5267a7-a185-4340-a053-49508e57c9cb)
![Artboard 1 copy 7-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/a47ad5ce-bb09-4186-8ee0-fd8fcdde9cb7)
![1 copy-100(1)](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/2af044d2-4d49-49e8-b8fc-ced428c98d86)
![1-100](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/3ee7b697-8f38-4b70-81ff-7cc03706ea2e)
![1](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/2a3da3e9-3abf-4ab2-924f-4b1b5468534b)
![2](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/97edb97b-223e-4b6e-a15b-f4077cc625f7)

**Reply by CesarAugusto316** [reply]
![image](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/56573509/86b7dc1b-d497-4019-8a9c-b6fff6fed1b7)

Hola Dani, necesito images en formato .png o .svg para logotipos ya que el fondo debe ser transparente, como la de aqui arriba. Por si acaso no tienes una que no tenga los bordes pixelados?

**Comment by dani97munoz** [comment]
![2](https://github.com/cometa-dani/cometa-mobile-app-ui/assets/147836235/fefdda1d-0f31-4a19-9323-8d8b0fbd97b5)

**Reply by CesarAugusto316** [reply]
Listo Dani. Muchas Gracias. Vamos que vamos
